packages/ui/cart/drawer/CartDrawer.tsx

```diff
@@ -85,7 +86,12 @@ export function CartDrawer() {
                               {cart.isLoading ? (
                                  <Spinner size="xs" />
                               ) : (
-                                 cart.data?.totals.itemsCount ?? 0
+                                 <>
+                                    <VisuallyHidden>
+                                       Number of Items in Cart:
+                                    </VisuallyHidden>
+                                    {cart.data?.totals.itemsCount ?? 0}
```

Comment:
Should this whole block be conditional based on `cart.data` being present?

Reply:
I think we intend to show `0` if there is no data